webrtc.html

```diff
@@ -16610,6 +16610,21 @@ <h4>
                   double rounded to the tenth decimal place.
                 </td>
               </tr>
+              <div id="remote-track-settings-channel-count">
+              <tr id="def-constraint-channelCount">
+                <th data-tests="RTCRtpEncodingParameters-codec-opus-stereo.https.html">
+                  channelCount
+                </th>
+                <td>
+                  {{ConstrainULong}}
+                </td>
+                <td>
+                  As a setting, this is the number of channels of the latest
+                  audio frame received, reflecting the number of
+                  {{RTCRtpCodec/channels}} that the decoder outputted.
```

Comment:
Comment from Youenn: make this more clear that you MUST do what the decoder is capable of